README.md

```diff
@@ -130,6 +130,8 @@ The `--fix` option on the command line automatically fixes problems reported by
 |    | Rule ID | Description |
 |:---|:--------|:------------|
 | :white_check_mark: | [avoid-leaking-state-in-ember-objects](./docs/rules/avoid-leaking-state-in-ember-objects.md) | Avoids state leakage |
+|  | [classic-decorator-hooks](./docs/rules/classic-decorator-hooks.md) | Prevent usage of classic APIs such as get/set in classes that aren't explicitly decorated with @classic |
+|  | [classic-decorator-no-classic-methods](./docs/rules/classic-decorator-no-classic-methods.md) | Prevent usage of classic APIs such as get/set in classes that aren't explicitly decorated with @classic |
```

Comment:
You need to run `yarn update` again to make sure these descriptions match what the rules specify.